extension/lib/bigcommerce/CartRepository.js

```diff
@@ -138,6 +138,23 @@ class BigCommerceCartRepository {
 
     return response.data.checkout_url
   }
+
+  /**
+   * @param {Array} cartItemIds
```

Comment:
This should rather be an array of strings.

Reply:
done